### DISCUSSION TITLE
New update has broken the colab

### DISCUSSION THREAD

**Discussion by 10XTMY** [TheLastBen/fast-stable-diffusion#2760] [General]
I don't know what changes were just made but my setup no longer works. I deleted everything and tried again now I get a constant directory does not exist for /repositories ??? that folder was not in previous setup and it is not in this one... I'm using the provided colab notebook. It was working perfectly last night now its broken. Please advise...

I have tried clean install three times same error every time. When cloning the repo there is no subdir /repositories after the pulls.
`
ControlNet preprocessor location: /content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads
2024-03-02 12:39:17,343 - ControlNet - INFO - ControlNet v1.1.440
2024-03-02 12:39:17,874 - ControlNet - INFO - ControlNet v1.1.440
Calculating sha256 for /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/sd_xl_base_1.0.safetensors: 2024-03-02 12:39:18,565 - ControlNet - INFO - ControlNet UI callback registered.
Running on public URL: https://ee049d51003c23e7df.gradio.live/
✔ Connected
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 162, in <module>
    webui()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 109, in webui
    ui.setup_ui_api(app)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui.py", line 1242, in setup_ui_api
    app.mount("/webui-assets", fastapi.staticfiles.StaticFiles(directory=launch_utils.repo_dir('stable-diffusion-webui-assets')), name="webui-assets")
  File "/usr/local/lib/python3.10/dist-packages/starlette/staticfiles.py", line 57, in __init__
    raise RuntimeError(f"Directory '{directory}' does not exist")
RuntimeError: Directory '/content/gdrive/MyDrive/sd/stable-diffusion-webui/repositories/stable-diffusion-webui-assets' does not exist
31e35c80fc4829d14f90153f4c74cd59c90b779f6afe05a74cd6120b893f7e5b
Loading weights [31e35c80fc] from /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/sd_xl_base_1.0.safetensors
Creating model from config: /content/gdrive/MyDrive/sd/stablediffusion/generative-models/configs/inference/sd_xl_base.yaml
Applying attention optimization: xformers... done.
Model loaded in 36.8s (calculate hash: 25.3s, load weights from disk: 3.3s, create model: 1.0s, apply weights to model: 6.1s, load textual inversion embeddings: 0.4s, calculate empty prompt: 0.4s).
Killing tunnel 127.0.0.1:7860 <> https://ee049d51003c23e7df.gradio.live/
`

**Comment by Kaka32793784** [comment]
Probably I'm Stable diffusionwo in Colab
I think it's the same reason why it can't be built.
Who should I contact?

**Comment by 10XTMY** [comment]
sorry this was meant to be in issues not discussion! I was in the wrong tab.
